### PR TITLE
ffmpeg6: strip static libs fix for i686-musl

### PR DESCRIPTION
**Pull request by xeroxslayer**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture: x86_64-musl
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64
  - i686
  - i686-musl
  - aarch64
  - aarch64-musl
  - armv7l
  - armv7l-musl

### CODE REVIEW

**Comment by ahesford** [comment]
Please don't open duplicate pull requests.

**Comment by xeroxslayer** [comment]
It's not a duplicate, one is for ffmpeg, the other for ffmpeg6, different templates.

**Comment by ahesford** [comment]
Sorry, my mistake.

File: srcpkgs/ffmpeg6/template [hunk 1]
```diff
@@ -196,7 +196,7 @@ ffmpeg6-devel_package() {
 	short_desc+=" - development files"
 	conflicts="ffmpeg-devel"
 	replaces="ffmpeg-devel>=0"
-	if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	if [[ "$XBPS_TARGET_MACHINE" = "i686"* ]]; then
```
Comment (ahesford) [inline srcpkgs/ffmpeg6/template:199]: As in the other template, please use a case block rather than the bash test operator here.

**Review by ahesford — CHANGES_REQUESTED** [review]

**Pull request closed without merging**